src/locales/en-us/index.ts

```diff
@@ -0,0 +1,134 @@
+        completeButtonLabel: 'Go to Zone “{name}”',
+        errorTitle: 'Could not connect Zone',
+        errorDescription: 'We were unable to connect Zone.',
+      },
+      universal: {
+        copySaveToken: {
+          title: 'Copy and save token',
+          alertMessage: 'You won’t see the token again! Make sure to save it somewhere now.',
+        },
+        connectZone: {
+          title: 'Connect Zone',
+          description1: 'Copy and paste the following configuration into the config.yaml on your local machine.',
+          fileName: 'config.yaml',
+          config: `environment: universal
+mode: zone
+multizone:
+  zone:
+    name: {zoneName}
+    globalAddress: {globalKdsAddress}
+kuma:
+  multizone:
+    zone:
+      kds:
+        auth:
+          cpTokenInline: {token}`,
+          command: 'kuma-cp run -c config.yaml',
+          description2: 'Next, run the following command to connect the Zone.',
+        },
+      },
+      kubernetes: {
+        prerequisites: {
+          title: 'Prerequisites',
+          step1Label: 'Kubernetes cluster',
+          step1LabelAddendum: 'with load balancer',
+          step1Description: '{productName} is compatible with all distributions of Kubernetes, including Minikube, AKS, EKS, GKE and OpenShift.',
+          step2Label: 'kubectl or oc access',
+          step2Description: 'You have kubectl or oc (if working with OpenShift) installed and configured to communicate to your Kubernetes cluster.',
+          step3LinkTitle: 'Helm 3',
+          step3Tail: 'is installed.',
+        },
+        helm: {
+          title: 'Set-up Helm',
+          description: 'On your local machine, create a namespace in your Kubernetes cluster and pull down the Helm repo.',
+          step1Description: 'Create a namespace',
+          step1Command: 'kubectl create namespace kuma-system',
+          step2Description: 'Add the charts repository',
+          step2Command: 'helm repo add kuma https://kumahq.github.io/charts',
+          step3Description: 'Update Helm',
+          step3Command: 'helm repo update',
+        },
+        secret: {
+          title: 'Set-up secret',
+          description1: 'This is your token.',
+          description2: 'Add it as a Kubernetes secret.',
+          command: `echo "
+apiVersion: v1
+kind: Secret
+metadata:
+name: cp-token
+namespace: kuma-system
+type: Opaque
+data:
+token: {token}
+" | kubectl apply -f -`,
+        },
+        connectZone: {
+          title: 'Connect Zone',
+          description1: 'Copy and paste the following configuration into the Helm chart values.yaml on your local machine.',
+          fileName: 'values.yaml',
+          config: `controlPlane:
+mode: zone
+zone: {zoneName}
+kdsGlobalAddress: {globalKdsAddress}
+secrets:
+  - Env: KUMA_MULTIZONE_ZONE_KDS_AUTH_CP_TOKEN_INLINE
+    Secret: cp-token
+    Key: token
+ingress:
+enabled: {zoneIngressEnabled}
+egress:
+enabled: {zoneEgressEnabled}`,
```

Comment:
**Issue**: Keys following `controlPlane:` need to be indented one level until and excluding `ingress`. Both `enabled` keys need to be indented one level.

```suggestion
  mode: zone
  zone: {zoneName}
  kdsGlobalAddress: {globalKdsAddress}
  secrets:
    - Env: KUMA_MULTIZONE_ZONE_KDS_AUTH_CP_TOKEN_INLINE
      Secret: cp-token
      Key: token
ingress:
  enabled: {zoneIngressEnabled}
egress:
  enabled: {zoneEgressEnabled}`,
```

Reply:
This has already been fixed in the followup PR https://github.com/johncowen/kuma-gui/compare/chore/core-i18n...johncowen:kuma-gui:chore/i18n-yaml . It'll be easier to just leave this as is here.

Reply:
Fine by me. Make sure to de-indent the `ingress` and `egress` blocks one level in that changeset though.